adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantContextConverter.scala

```diff
@@ -313,6 +315,36 @@ private[adam] class VariantContextConverter(dict: Option[SequenceDictionary] = N
   }
 
   /**
+   * Split the htsjdk variant context ID field into an array of names.
+   *
+   * @param vc htsjdk variant context
+   * @return Returns an Option wrapping an array of names split from the htsjdk
+   *    variant context ID field
+   */
+  private def splitIds(vc: HtsjdkVariantContext): Option[java.util.List[String]] = {
```

Comment:
Can you write some unit tests for this? Ideally, we'd have tests for just this function with a VC with no ID, one ID, more than one ID, and an end-to-end VCF-with-IDs test.